libvirt.go

```diff
@@ -229,11 +226,14 @@ func (d *Driver) PreCreateCheck() error {
 }
 
 func (d *Driver) Create() error {
-	if !strings.HasPrefix(d.DiskPathURL, filePrefix) {
-		return fmt.Errorf("DiskPathURL must start with %s", filePrefix)
+	if d.ImageFormat == "" {
```

Comment:
I still think this is not needed since we should always ask the `ImageFormat` when constructing the driver like in the hyperkit side https://github.com/code-ready/machine-driver-hyperkit/pull/19/files#diff-58e2eb65dc92592158d33ad1c695febcR222-R224

Reply:
I tried removing it, and had some issues,  I'll try again :)